src/sortStudents.ts

```diff
@@ -1,16 +1,54 @@
-
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
+
+function sorted(
+  studentArray: Student[],
+  sortBy: SortType,
+  order: string,
+): object[] {
+  return studentArray
+    .sort((student1: Student, student2: Student) => {
+      const output = typeof student1[sortBy] === 'string'
+        ? student1[sortBy].toString().localeCompare(student2[sortBy].toString())
```

Comment:
why do you need this is you already checked type of `student1[sortBy]`
```suggestion
        ? student1[sortBy].localeCompare(student2[sortBy].toString())
```